src/wallet/load.cpp

```diff
@@ -90,7 +90,7 @@ bool VerifyWallets(interfaces::Chain& chain)
     return true;
 }
 
-bool LoadWallets(interfaces::Chain& chain)
+bool LoadWallets(interfaces::Chain& chain, const std::unique_ptr<interfaces::CoinJoin::Loader>& coinjoin_loader)
```

Comment:
don't pass const ref to unique_ptr; pass a const ref to the underlying

Reply:
see https://github.com/dashpay/dash/pull/5767#discussion_r1446359310